og.module

```diff
@@ -47,8 +47,11 @@ function og_entity_insert(EntityInterface $entity) {
     return;
   }
 
-  $membership = Og::createMembership($entity, $entity->getOwner());
-  $membership->save();
+  // Other modules might interfere and create a membership.
+  if (!Og::getMembership($entity, $entity->getOwner())) {
```

Comment:
I think here you will get only the `active` memberships.

Reply:
Indeed!! I have also been trapped by this in the past, which is why I created #292 to lobby for changing this to return all memberships by default. I think it is not intuitive that this method returns filtered results by default, and an additional parameter should be passed to get the actual membership.